### ISSUE TITLE
Allow duplicate custom parameters

### ISSUE THREAD

**Issue by khaledhosny** [googlefonts/glyphsLib#672]
Glyphs UI seems to allow it, and Glyphs will happily build the font (the last custom parameter seems to win), but glyphsLib will raise:
```
  File "/venv/lib/python3.9/site-packages/glyphsLib/builder/custom_params.py", line 106, in get_custom_value
    raise RuntimeError(f"More than one value for this customParameter: {key}")
RuntimeError: More than one value for this customParameter: Use Typo Metrics
```

**Comment by IvanUkhov** [comment]
This should be addressed. At least for the naming table. See #827.

**Comment by khaledhosny** [comment]
Thanks, closing.

**Comment by khaledhosny** [comment]
> @khaledhosny you closed this one but #903 is still pending?

Different issues, this one was about the builder and I don’t remember how to reproduce it, #903 is about GSFont class losing data, GSInstance and GSFontMaster has no issue.
